test/integration/index_management.test.js

```diff
@@ -258,39 +271,63 @@ describe('Indexes', function () {
     }
   });
 
-  it('shouldCorrectlyDropIndexes', {
-    metadata: {
-      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
-    },
+  context('resolved dropIndexes', function () {
```

Comment:
`resolved dropIndexes` describes the code (it uses promises now, it didn't before), not the behavior.  tests should describe the behavior under test.

this applies to all the tests here
For example:
```typescript
context('when dropIndexes succeeds', function() {
	it('dropIndexes returns true', function() {

	});

	it('the index is no longer present on the collection', function () {

	})
})
```

Reply:
I can't separate the two tests here because of the beforeEach block